### PR TITLE
Add patch for ACM 2.10 and 2.11

### PR DESCRIPTION
**Pull request by tingxueca**

### CODE REVIEW

File: patch/acm-2.10.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#!/usr/bin/env bash
```
Comment (akrzos) [inline patch/acm-2.10.sh:2]: nitpick, there is an extra line here

File: patch/acm-2.11.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#!/usr/bin/env bash
```
Comment (akrzos) [inline patch/acm-2.11.sh:2]: ditto

**Review by akrzos — CHANGES_REQUESTED** [review]
Extra sha-bang on both scripts, lets remove them so we can merge this.  Thanks @tingxueca

File: patch/acm-2.10.sh (hunk 1 above)
Comment (tingxueca) [reply patch/acm-2.10.sh]: fixed

File: patch/acm-2.11.sh (hunk 2 above)
Comment (tingxueca) [reply patch/acm-2.11.sh]: fixed

**Review by akrzos — APPROVED** [review]
LGTM

**Pull request merged**